File: C#/TwoSum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        
        int[] output = {0, 0};
        
        for (int i = 0; i < nums.Length; i++)
        {
            for (int j = 0; j < nums.Length; j++)
            {
                if (i != j)
                {
                    if (nums[i] + nums[j] == target)
                    {
                        output[0] = i;
                        output[1] = j;
                        return output;
                    }
                }
            }
        }
        
        return null;
    }
}

File: Python/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        for i in range(0, len(nums)):

            diff = target - nums[i]

            try:
                index = nums.index(diff)
            except:
                index = -1
            else:
                if i != index:
                    print(i,index)
                    return [i,index]

        return [-1,-1]